ui/src/app/components/sidenav-review-changes/sidenav-review-changes.component.scss

```diff
@@ -0,0 +1,33 @@
+.container {
```

Comment:
Please use the styles.css for these as most of the css related to container has been defined there and anything specific can be grouped under related class and placed in styles.css

Reply:
updated